helm/polaris/templates/deployment.yaml

```diff
@@ -90,11 +90,11 @@ spec:
               mountPath: {{ .Values.logging.file.logsDir }}
               readOnly: false
             {{- end }}
-            - name: temp-dir
```

Comment:
I think you accidentally deleted some volume mounts here. Also, there is a `polaris.containerPorts` template that prints the ports, you don't need to iterate over the ports here.